strawberryfields/apps/points.py

```diff
@@ -41,11 +41,22 @@
 semidefinite, it is possible to use GBS to implement a *permanental* point process and employ
 fast classical algorithms to sample from the resulting distribution.
 
+One choice of kernel matrix is the radial basis function (RBF) kernel whose elements are computed
```

Comment:
@soranjh this involved a bit of a change to the docstrings, just wanted to check if you think this is ok?

Reply:
All good.